docs/addons/writing-addons.md

```diff
@@ -97,11 +97,13 @@ Going through the code blocks in sequence:
 // src/Tool.tsx
 
 import { useGlobals, useStorybookApi } from '@storybook/manager-api';
-
-import { Icons, IconButton } from '@storybook/components';
+import { IconButton } from '@storybook/components';
+import { LightningIcon } from '@storybook/icons';
 ```
 
-The [`useGlobals`](./addons-api.md#useglobals) and [`useStorybookApi`](./addons-api.md#usestorybookapi) hooks from the `manager-api` package are used to access the Storybook's APIs, allowing users to interact with the addon, such as enabling or disabling it. The `Icons` and `IconButtons` components from the [`@storybook/components`](https://www.npmjs.com/package/@storybook/components) package render the icons and buttons in the toolbar.
+The [`useGlobals`](./addons-api.md#useglobals) and [`useStorybookApi`](./addons-api.md#usestorybookapi) hooks from the `manager-api` package are used to access the Storybook's APIs, allowing users to interact with the addon, such as enabling or disabling it.
```

Comment:
@cdedreuille — I pushed [the commit](https://github.com/storybookjs/storybook/pull/25822/commits/997fe1161a19b6bececf4b4b2a2caa86fffd32ed) with further updates.